lib/modules/customToggles.js

```diff
@@ -50,6 +52,20 @@ function anyTogglesEnabled(toggles) {
 		.some(enabled => enabled);
 }
 
+function registerCommandLine() {
+	CommandLine.registerCommand(/^tog$|^toggle$/, ' tog[gle] [name] - toggles a custom toggle',
+		() => {},
```

Comment:
It would be nice to [match partial names](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/5d76b584ac180e36208ceb3939c2c08822900de6/lib/modules/settingsConsole.js#L71), and see what action this perform when executed (i.e. will this activate / disable the toggle).

Reply:
Fixed! Btw, I noticed that the help message will only show if the helper function returns `null`. So in the module command it will never show. See my updated code